plugins/factory-plugin/src/workspace-projects-manager.ts

```diff
@@ -212,7 +208,7 @@ export class WorkspaceConfigProjectsManager extends WorkspaceProjectsManager {
 
         projectsHelper.deleteProjectFromWorkspaceConfig(
             workspace.config.projects,
-            fileUri.convertToCheProjectPath(projectFolderURI, this.projectsRoot)
+            '/' + fileUri.convertToCheProjectPath(projectFolderURI, this.projectsRoot)
```

Comment:
Maybe  `path.sep` here instead of '/'?  But up to you.

Reply:
According to [specification](https://github.com/eclipse/che/blob/e3281718a71db9f4de913d597dea0107f6c4b20f/core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/devfile/Project.java#L23), only `/` is allowed.